src/routes/version_creation.rs

```diff
@@ -70,8 +70,8 @@ pub fn check_version(version: &InitialVersionData) -> Result<(), CreateError> {
     Ok(())
 }
 
-// under `/api/v1/mod/{mod_id}`
-#[post("version")]
+// under `/api/v1/version`
+#[post("")]
```

Comment:
This may not work the way you expect it to.  In every other place that we do this, the route is specified as `#[post("version")]` and is outside of the `version` scope: https://github.com/modrinth/labrinth/blob/f3c61870dcc80620ade00193e44b7d179742b2e1/src/routes/mod.rs#L21-L30